### ISSUE TITLE
Add-on don't work in NVDA 2021.1 Beta1

### ISSUE THREAD

**Issue by ruifontes** [jscholes/nvda-speech-history#17]
To adapt it to new NVDA is necessary the following:
1 - In manifest.ini change the "lastversiontested" entry;
2 - In SpeechHistory.py it is necessary to change:
oldSpeak = speech.speak
to:
oldSpeak = speech.speech.speak
3 - Change
        speech.speak = self.mySpeak
to:
		speech.speech.speak = self.mySpeak
4 - Change:
		speech.speak = oldSpeak
to:
		speech.speech.speak = oldSpeak
5 - By the way, can you prepare the add-on to be translated?

**Comment by fisher729** [comment]
In addition, Speech History does not work well when "Attempt to cancel speech for expired focus events" is set to "yes", which is the default in NVDA 2021.1.

**Comment by jscholes** [comment]
Closed via https://github.com/jscholes/nvda-speech-history/commit/c7e11aaa58815c38370a4a8f0b7033700136956a.
